src/stories/Quiz/Examples/index.stories.tsx

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable no-console, react/jsx-props-no-spreading */
 import CioQuiz from '../../../components/CioQuiz';
-import { argTypes } from '../argTypes';
+import { argTypes, storiesControls } from '../argTypes';
 
 import RetrievingAnswersStory from './RetrievingAnswers';
 
 export default {
   title: 'Quiz/Specific Examples',
   component: CioQuiz,
```

Comment:
It looks like the controls on the `Specific Examples/Docs` page doesn't seem to be doing anything. Is it possible to fix them or should we disable the controls?

Reply:
Better to be removed